src/views/Jobs/JobDetails.vue

```diff
@@ -40,6 +40,21 @@
                     ></v-progress-circular>
                   </div>
                 </v-list-tile>
+                <v-list-tile class="ma-3">
+                  <v-icon class="mr-2">cached</v-icon>
+                  <div class="body-1">
+                    Conditions:<br /><strong v-if="prediction">{{
+                      prediction.aerobic ? "Aerobic" : "Anaerobic"
```

Comment:
You can use `job.aerobic` without waiting for predictions to finish processing (just like in the Jobs table)